lmdeploy/messages.py

```diff
@@ -112,7 +112,7 @@ class TurbomindEngineConfig:
     tp: int = 1
     session_len: Optional[int] = None
     max_batch_size: int = 128
-    cache_max_entry_count: float = 0.5
+    cache_max_entry_count: float = 0.8
```

Comment:
We should set the default value at https://github.com/InternLM/lmdeploy/blob/main/lmdeploy/turbomind/deploy/target_model/base.py#L58 too. cc @AllentDan

Reply:
Hi @lzhangzz Besides that, we may update this default value too. cc @AllentDan 
https://github.com/InternLM/lmdeploy/blob/b59da12ef5e49c0ed141d336c48b21a4b82b46a4/lmdeploy/cli/utils.py#L308-L316